### DISCUSSION TITLE
Making it easy to link to headings in other notes

### DISCUSSION THREAD

**Discussion by Colin5000** [tadashi-aikawa/obsidian-various-complements-plugin#294] [Ideas]
I'd love to be able to type `Chapter 5` to get `[[Big Document#Chapter 5]]` without ever typing `Big Document#`. 

Possible?

On a somewhat related, er, note, as a recent fan of the Omnisearch plugin I wonder whether it could be used as the search engine by Various Complements. It's already good at finding headings.

**Comment by tadashi-aikawa** [comment]
Hi, @Colin5000 :)

Is [this issue](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/72) the same as the one you're referring to? If so, I would appreciate it if you could watch that issue 👍

**Comment by Colin5000** [comment]
Yes, that's exactly it! It seemed like too obvious a feature for me to be
the only person to think of it :)

Looking forward to it. Thank you so much for Various Complements - it's a
great idea, and beautifully executed.

**Reply by MadaboutDana** [reply]
It’s an astonishing idea! And really well executed (fast, unobtrusive, comprehensive...)
